packages/codelabs/src/basics/lit-html.md

```diff
@@ -1558,10 +1554,10 @@ class TodoList extends LitElement {
 customElements.define('todo-list', TodoList);
 ```
 
-The styles we define here only apply to our element. This is a guarantee built into the browser because we're using shadow DOM, lit-element does not need to do any extra work for that. This means we can write simple CSS selectors, and not need to worry about creating conflicts with styles defined elsewhere on the page.
+The styles we define here only apply to our element. This is guaranteed, because we are using shadow DOM the is built in the browser and lit-element does not need to do any extra work. This means we can write simple CSS selectors, and we don't need to worry about causing conflicts with styles defined elsewhere on the page.
```

Comment:
```suggestion
The styles we define here only apply to our element. This is guaranteed, because we are using shadow DOM which is built in the browser and because of that, lit-element doesn't need to do any extra work. This means we can write simple CSS selectors, and we don't need to worry about causing conflicts with styles defined elsewhere on the page.
```